src/app/accounts/disable-notification-channel.ts

```diff
@@ -0,0 +1,23 @@
+import { setNotificationChannelIsEnabled } from "@domain/notifications"
+import { AccountsRepository } from "@services/mongoose"
+
+export const disableNotificationChannel = async ({
+  accountId,
+  notificationChannel,
+}: {
+  accountId: AccountId
+  notificationChannel: NotificationChannel
+}): Promise<Account | ApplicationError> => {
+  const account = await AccountsRepository().findById(accountId)
+  if (account instanceof Error) return account
+
+  const newNotificationSettings = setNotificationChannelIsEnabled({
```

Comment:
name not very consistent with `src/app/accounts/disable-notification-category.ts` where the main action is `domainDisableNotificationCategory` (ie: not starting with set)

Reply:
I will make the names consistent